include/arch/arm/arch/64/mode/object/structures.bf

```diff
@@ -144,8 +144,8 @@ block cb_control_cap {
 block cb_cap {
 
     padding               40
-    field capBindSID      12
-    field capCB           12
+    field capBindSID      16
+    field capCB           8
```

Comment:
You can't change the size of the CB field in only one cap type and structure -- other places in the kernel are expecting the CB field to be 12 bits wide, so if this is really necessary it would need a wider change and validation. There is padding available, so it looks like this could be left at 12, but I'm curious why this is being changed.

Reply:
The reasoning here goes back to the SMMUv2 spec just like the bit changes for the SID cap. The spec
states a maximum of 128 context banks so 12 bits is in excess of this maximum. So this could be
dropped if need be. I am not seeing off hand where the dependency on the capCB field being 12 bits is
in the rest of the kernel. Could you point me to where it is that you are referencing?

Reply:
Ah I think I found it, are you referencing `capPUDMappedCB` and similar bit fields?

Reply:
Yes, thanks for looking, those are the ones I meant.

Reply:
(If we do the the approach that we follow the maximum allowed by the spec, we should identify and change all of these)

Reply:
Change for CB bits already in latest master